### DISCUSSION TITLE
Aorus Z690

### DISCUSSION THREAD

**Discussion by milkywayco** [Rem0o/FanControl.Releases#949] [General]
Anyone know what to set in bios for smart fan 6 on the Z690? My fan control detects the speeds of the fans but isn't able to control any of them. I've gotten it to have limited control when I turn each fan header to "full speed" and "pwm" in bios but that only lets me bring down the speed to about 900rpm (no lower) and only on sys fan 2 and 3. CPU and sys fan 4 continue to ignore controls from fan control. None of the fans are in the pump headers.

Any ideas?

**Comment by Migratingcocofruit** [comment]
Try and tick "the force apply" option from the 3 dot menu on the fan card.

![image](https://user-images.githubusercontent.com/69551563/189164249-3367b464-d4c9-4035-beb6-a255bfaa709c.png)

**Comment by Velocitas77** [comment]
What does Force apply exactly do different? Thank you

**Reply by Rem0o** [reply]
Force apply will try to force set the value every update cycle. Won't help with compatibility in the first place.

**Reply by Velocitas77** [reply]
Thank you for the reply. I was asking because my ASUS setup does have problems with zero rpm. Below a certain threshold my fans will stop but for some reason they dont turn on again. RPM shows "?" and what helps is refresh detection. First time I realized that, my water was almost at 60 degrees. When I set a minium rpm to remain silent, no problems. Only with zero RPM.
